### PR TITLE
BaSP-TG-78: Login styles

### PR DESCRIPTION
**Pull request by DinoStampella**
image

### CODE REVIEW

File: src/Components/Auth/LogIn/index.js [hunk 1]
```diff
@@ -21,7 +23,7 @@ const Login = () => {
     resolver: joiResolver(loginSchema),
     mode: 'onChange'
   });
-
+  console.log('Errors: ', errors);
```
Comment (JulianFloresDev) [inline src/Components/Auth/LogIn/index.js:26]: Delete consoles!

File: src/Components/Auth/LogIn/index.js [hunk 2]
```diff
@@ -5,8 +5,10 @@ import { loginSchema } from 'Validations/loginSchema';
 import { useDispatch, useSelector } from 'react-redux';
 import { login } from 'redux/auth/thunks';
 import { useHistory } from 'react-router-dom';
-import { InputForm, Modal, Spinner } from 'Components/Share';
+import { Modal, Spinner } from 'Components/Share';
 import { setModalContent, setShowModal } from 'redux/global/actions';
+import styles from './login.module.css';
+import BackArrow from 'Components/Share/BackArrow/backArrow';
```
Comment (JulianFloresDev) [inline src/Components/Auth/LogIn/index.js:11]: It should be imported from 'Component/Share' make sure this is correctly exported from there!

File: src/Components/Auth/LogIn/login.module.css [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  text-align: center;
+  height: 100vh;
+}
+
+.loginForm {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  width: 35%;
+  padding: 5%;
```
Comment (JulianFloresDev) [inline src/Components/Auth/LogIn/login.module.css:14]: Same as padding, try use absolute units, lik3 20px or 1rem.

File: src/Components/Auth/LogIn/login.module.css [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+}
+
+.loginHeader {
+  display: flex;
+}
+
+.loginHeader > figure {
+  display: flex;
+  align-items: center;
+  width: 15%;
```
Comment (JulianFloresDev) [inline src/Components/Auth/LogIn/login.module.css:25]: Something similar happens here, when the width of the screen gets smaller and smaller, the arrow gets smaller and smaller to the point of not being visible.

File: src/Components/Auth/LogIn/login.module.css [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+  justify-content: space-between;
+  font-size: 12px;
+}
+
+.loginFunctionalities > div {
+  display: flex;
+  align-items: center;
+  gap: 5%;
+  width: 50%;
+}
```
Comment (JulianFloresDev) [inline src/Components/Auth/LogIn/login.module.css:61]: I suggest use `flex-wrap: wrap` property.

When viewport width is less than 700px it should be one property under the other.

image

File: src/Components/Auth/LogIn/index.js (hunk 2 above)
Comment (DinoStampella) [reply src/Components/Auth/LogIn/index.js:11]: At least, in my local repo, the folder is called Components. We should check that all of us have the same folder name.

File: src/Components/Auth/LogIn/login.module.css (hunk 5 above)
Comment (DinoStampella) [reply src/Components/Auth/LogIn/login.module.css:61]: I add flex-wrap property but now that its container has absolut measures it doesn't collapse. Anyway, now it's better

**Review by kolodani — DISMISSED** [review]
Nice work!!!

**Review by JulianFloresDev — DISMISSED** [review]
Well Done!! 

I suggest add some features in red modal close button ( X ), just for be consistant. But in my opinion it can be merge!!

**Comment by anapaulib** [comment]
Well done, guys!!
I like that error messages appear immediately. Just a little correction. To maintain some coherence it would be better if the form's title is Log In as same as the button.

**Review by MatiasPazConti — APPROVED** [review]

**Review by kolodani — APPROVED** [review]

**Review by agusgatto8 — APPROVED** [review]

**Pull request merged**